src/ftest.jl

```diff
@@ -85,29 +94,25 @@ function ftest(mods::LinPredModel...)
         pvals[i] = result.pval
     end
 
-    return CoefTable([SSR1s, SSR2s, df1s, df2s, MSR1s, MSR2s, fstats, pvals],
-    ["Model 1 SSR", "Model 2 SSR", "Model 1 df", "Model 2 df", "Model 1 MSR", "Model 2 MSR", "F statistic", "p-value"],
-    ["Model $(i-1):$i" for i in 2:nmodels])
+    return FTestTable(SSR1s, SSR2s, df1s, df2s, MSR1s, MSR2s, fstats, pvals)
 end
 
 ### Utility functions to show FTestResult and MultiFTestResult
-function show(io::IO, x::FTestResult)
+function show(io::IO, x::FTestTable)
```

Comment:
Actually, looking at the equivalent code for `CoefTable`, I realize it's a bit complex to print this kind of tables correctly. So maybe after all maybe we should reuse `CoefTable` here? If so we should rename it later to make it more general.